cypress/support/commands.ts

```diff
@@ -62,15 +62,6 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add("login", () => {
-  cy.wait(4000); //prevent random failure
-  cy.get("#rh-password-verification-submit-button").then(($item) => {
-    if ($item.is(":visible")) {
-      cy.log("Cookie's choise is not present");
-    } else {
-      cy.log("Cookie's choise is present");
-      cy.get("button:contains('Accept default')").click().should("not.exist");
-    }
-  });
```

Comment:
I agree with removing this part of login command. 

I was not able to find out an element, that would match `#rh-password-verification-submit-button` selector. To me it seems like an outdated?

Reply:
I thought this part was responsible for closing the cookie pop-up. Is it covered elsewhere now? Or isn't it necessary anymore?

Sorry, I am kinda lost. This PR title and description don't help much understanding what's going on.

Reply:
@kelvah Unfortunately it is not cover right now. I can not reproduce it locally so I have not got the html code. The condition does not work properly. The problem still occurs in GHAs.

Reply:
@sjamboro Thanks for the explanation.